package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "repository": "googleapis/nodejs-showcase",
+  "name": "showcase",
+  "version": "0.1.0",
+  "author": "Google LLC",
+  "description": "GAPIC Showcase client for Node.js",
+  "main": "build/src/index.js",
+  "files": [
+    "protos",
+    "build/src",
+    "!build/src/**/*.map",
+    "AUTHORS",
+    "COPYING"
+  ],
+  "dependencies": {
+    "google-gax": "googleapis/gax-nodejs"
+  },
+  "devDependencies": {
+    "@types/mocha": "^5.2.5",
+    "@types/through2": "^2.0.34",
```

Comment:
We really shouldn't need `through2` after node6 goes EOL - it's just a `PassThrough` stream.

Reply:
I will try removing it and will see what breaks :)